src/sentry/services/sentryshot/service.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from sentry.models.sentryshot import SentryShot
+
+MAX_WAIT_TIME = 5
+logger = logging.getLogger("service:sentryshot")
+
+
+class SentryShotService:
+    def __init__(self, uuid):
+        self.sentryshot = SentryShot.objects.filter(uuid=uuid).first()
+
+    def _start(self):
+        chrome_options = Options()
+        chrome_options.add_argument("--headless")
+        self.driver = webdriver.Chrome(options=chrome_options)
```

Comment:
Everything here ideally will only run once per instance so it doesn't matter. But`generate_screenshot` will be retried from the outside call, and in case the reason of failure was that webdriver timeout or something I want to include this in the retry.